codalab/lib/cli_util.py

```diff
@@ -71,11 +72,19 @@ def parse_key_target(spec):
         - <key>: (<key> if present,
                     empty string if ':' in spec but no <key>,
                     None otherwise)
-        - <value> (where value is everyhing after a <key>: (or everything if no key specified)
+        - <value> (where value is everything after a <key>: (or everything if no key specified)
     """
-
     match = re.match(TARGET_KEY_REGEX, spec)
-    return match.groups()
+    key, value = match.groups()
+    # This check covers three usage errors:
```

Comment:
I don't think we need this comment -- it seems pretty clear that this is checking when `value` is empty.

Reply:
Since this logic covers some edge cases that weren't put into consideration before. I think it would be helpful to have this documented.